vaccine/vaccine_reg_ussd.py

```diff
@@ -406,16 +406,23 @@ async def state_municipality(self):
         if not required:
             return await self.go_to_state("state_suburb")
 
+        async def next_state(choice: Choice):
+            if choice.value == "other":
+                return "state_suburb_search"
+            return "state_suburb"
+
         question = "Please select your municipality"
+        choices = [Choice(k, v[:30]) for k, v in results]
         choices = enforce_character_limit_in_choices(
-            [Choice(k, v[:30]) for k, v in results], 160 - len(question)
+            choices, 160 - len(question) - len("X. Other ")
```

Comment:
I think we'll need to take into account 2 newlines here, one after the question, and one between the choices and the other choice

Reply:
I added an extra space after the "Other" option for the first linebreak.. I'll add another for the linebreak after the question